docs/process/teams.md

```diff
@@ -0,0 +1,28 @@
+# Teams
+
+**Who do you `@` for what? See below!**
+
+_(These teams are not set in stone and will likely evolve as we continue to iterate on what makes the most sense for the success of the product.)_
+
+## General Teams
+
+These are good teams to start with for general communication and questions. (Members of these teams are also members of the skills-based teams below.)
+
+| Team | Purpose |
+|:--|:--|
+| `@desktop/maintainers` | The people designing, developing, and driving GitHub Desktop |
```

Comment:
I think it'd still be helpful to know which groups `@desktop/maintainers` includes so that I don't mention extra groups, ex: `@desktop/maintainers` **with** `@desktop/comrades` if i don't have to. I think we could just add in a clause that maintainers includes all of the groups below.